MatrixSDK/Utils/MXHTTPClient.m

```diff
@@ -81,6 +86,17 @@ -(id)initWithBaseURL:(NSString *)baseURL andAccessToken:(NSString *)access_token
         
         httpManager = [[AFHTTPRequestOperationManager alloc] initWithBaseURL:[NSURL URLWithString:baseURL]];
         
+        // Check whether some certificates have been included in app bundle
+        AFSecurityPolicy *securityPolicy = [AFSecurityPolicy policyWithPinningMode:AFSSLPinningModeCertificate];
+        if (securityPolicy.pinnedCertificates.count)
```

Comment:
This code looks only for certificate in app bundle (no in Documents dir).

I keep it here to support the original AFNetworking pinned certificates handling. This is an additional option for the developer to handle certificates